introduction.md

```diff
@@ -3,4 +3,10 @@ My name is Stian Henriksen
 My Linkedin page:
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?